libc/arc4random.c

```diff
@@ -13,8 +14,6 @@
 unsigned int arc4random(void)
 {
     unsigned int r;
-
-    while (!_rdrand32_step(&r))
-        ;
+    r = (unsigned int)_rdrand();
```

Comment:
@vishnumohan1991 
FYI, we are removing system includes from compiler include path when building enclaves.
So when #1052 goes in, immintrin.h will not be available. However you can use GCC's builtins where appropriate.